pennylane/devices/default_qutrit_mixed.py

```diff
@@ -67,6 +73,8 @@ def accepted_sample_measurement(m: qml.measurements.MeasurementProcess) -> bool:
     return isinstance(m, qml.measurements.SampleMeasurement)
 
 
+@simulator_tracking
+@single_tape_support
```

Comment:
simulator_tracking and single_tape_support modifiers are added. 

I am not sure if I should add simulator tracking as the code that computes shots is not necessarily true as it doesn't take into account commuting GellMann measurements, instead it counts it naively. This is similar to how the code is run on the device, but may not be the best for an actual device as you may want to be more careful with the samples taken.

If anyone has an opinion on if I should rewrite this, remove it, or leave it as is I would appreciate the input.

Reply:
@albi3ro could you help out here? I'm still a bit unfamiliar with how the tracker works.

Reply:
Right, it's a question of whether we are tracking for what is actually happening vs what we would want to happen on an optimized hardware device.

We have a similar question in `default.qubit` and solved it in https://github.com/PennyLaneAI/pennylane/pull/4628, with `executions` being the number of hardware executions and `simulations` being the number of times you would simulate.

So in this case, I guess we are correctly reporting `simulations` but `executions` could be lower?

Overall though, I think we should not block this PR - we can create an issue to follow up later.